actions/websocket_actions.jsx

```diff
@@ -1127,3 +1148,39 @@ function handleOpenDialogEvent(msg) {
 
     store.dispatch(openModal({modalId: ModalIdentifiers.INTERACTIVE_DIALOG, dialogType: InteractiveDialog}));
 }
+
+function handleGroupUpdatedEvent(msg) {
+    const data = JSON.parse(msg.data.group);
+    store.dispatch({
+        type: GroupTypes.RECEIVED_GROUP,
+        data,
+    });
+}
+
+function handleGroupAssociatedToTeamEvent(msg) {
+    store.dispatch({
+        type: GroupTypes.RECEIVED_GROUP_ASSOCIATED_TO_TEAM,
+        data: {teamID: JSON.parse(JSON.stringify(msg.broadcast.team_id)), groups: [{id: msg.data.group_id}]}
```

Comment:
I guess I'm a little confused here but why do we do JSON.parse(JSON.stringify(...)) ? 
Isn't it valid to just do a `teamID: msg.broadcast.team_id` ?

Reply:
Yes, I removed the unnecessary transformation, was leftover from a previous iteration.